lra/index.html.haml

```diff
@@ -0,0 +1,42 @@
+---
+layout: rightcol
+rightcol_partial: rightcol-documentation.html.haml
+title: Long Running Actions (LRA)
+documentation_highlight: true
+---
+
+.row
+  .col-md-12
+    %h1 Long Running Actions (LRA)
+    %p
+      Long Running Actions (abbreviated as
+      %code
+        LRA
+      ) is a MicroProfile specification that introduces APIs for services to coordinate activities.
+      For more information check site
+      %a{:href => "https://github.com/eclipse/microprofile-lra"} https://github.com/eclipse/microprofile-lra
+      \.
+    %p
+      Narayana implements the LRA specification.
+
+
+.row
+  .col-md-12
+    %h2
+      Certifications
+    %ul
+      %li
+        %a{:href => relative("/certifications/microprofile-lra-1.0/index.html")} LRA 1.0 Compatible Certification Request (CCR)
+
+.row
+  .col-md-12
+    %h2 Narayana LRA Coordinator OpenAPI Schema
+    %ul.list-group
+      %li.list-group-item
+        %a{:href => relative("/lra/api/1.0/openapi.yaml")}
+          %i.icon-external-link
+          Version: 1.0, format: YAML
+        %br
+        %a{:href => relative("/lra/api/1.0/openapi.json")}
```

Comment:
@ochaloup I think because it is relative this should be something like /api rather than /lra